server/src/coreV2/refresh-token/hooks/before-create-one-refresh-token.hook.ts

```diff
@@ -0,0 +1,24 @@
+import {
+  BeforeCreateOneHook,
+  CreateOneInputType,
+} from '@ptc-org/nestjs-query-graphql';
+import { v4 as uuidv4 } from 'uuid';
+
+import { RefreshToken } from 'src/coreV2/refresh-token/refresh-token.entity';
+
+export class BeforeCreateOneRefreshToken<T extends RefreshToken>
+  implements BeforeCreateOneHook<T, any>
+{
+  async run(
+    instance: CreateOneInputType<T>,
+    context: any,
+  ): Promise<CreateOneInputType<T>> {
+    const userId = context?.req?.user?.user?.id;
+
+    instance.input.userId = userId;
+    // FIXME: These fields should be autogenerated, we need to run a migration for this
+    instance.input.id = uuidv4();
+    instance.input.updatedAt = new Date();
```

Comment:
`id` and `updatedAt` are not autogenerated as they should be. I believe we need to run migrations for this in prisma(we're not yet running migrations for TypeOrm but just creating replica models that are in sync with the ones in Prisma)

#### Current `id`
<img width="759" alt="Screenshot 2023-11-08 at 15 28 13" src="https://github.com/twentyhq/twenty/assets/140154534/9045f486-09fd-428a-ab2d-208305e4a0a2">

#### Current `updatedAt`
<img width="759" alt="Screenshot 2023-11-08 at 15 28 07" src="https://github.com/twentyhq/twenty/assets/140154534/193f69be-eefa-444c-87e7-a08c3ef2cb01">